docker/lightgbm-ci-build-env/Dockerfile

```diff
@@ -60,8 +64,22 @@ RUN apt-get update \
     unzip \
     wget \
     zip \
+    make \
+    tar \
  && rm -rf /var/lib/apt/lists/*
 
+# Update ca certificates to safely perform curls
+RUN mkdir -p ~/.ca-certificates-src \
+    && cd ~/.ca-certificates-src \
+    && wget https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/ca-certificates/20210119~20.04.2/ca-certificates_20210119~20.04.2.tar.xz \
```

Comment:
for security reason, consider downloading the file.
the red flag is that you are executing `sudo make and sudo make install`

Reply:
Great point. fixed